README.md

```diff
@@ -4,22 +4,53 @@
 env name|required|default|description
 --------|--------|-------|-----------
 DATALAKE_CHANNEL_ID|True|None|DataLake channel id
-DATALAKE_TRAIN_ITEM_ID|True|None|DataLake item id
-DATALAKE_TEST_ITEM_ID|False|None|DataLake item id
+DATALAKE_TRAIN_FILE_ID|True|None|DataLake item id
```

Comment:
seems better to fix `DataLake item id` too.

Reply:
Indeed! 🙇